ecosystem/sep-0010.md

```diff
@@ -188,6 +191,12 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+## Validating control of a Stellar account
+
+Servers that receive a JWT that proves the holder has control of a secret key must take additional steps to verify that the holder of that secret key has control of a Stellar account. Even if the secret key is the master key for an account it may not have control of an account. For operations relating to a Stellar account a server on receipt of a JWT should verify in addition to the JWT's general validity that:
+
+- the public key in the `sub` has signing authority of the Stellar account at a sufficient threshold.
```

Comment:
Maybe replace authority with `weight` to match the [multi-sig docs](https://www.stellar.org/developers/guides/concepts/multi-sig.html)?